ptd_client_server/lib/client.py

```diff
@@ -118,20 +114,56 @@ def create_zip(zip_filename: str, dirname: str) -> None:
                 zf.write(filePath, zipPath)
 
 
-def reorder_loadgen_files(path: str) -> None:
+def get_time_from_line(
+    line: str, data_regexp: str, file: str, timezone_offset: int
+) -> Optional[float]:
+    # TODO: deduplicate with check.py?
```

Comment:
We had code sharing between the checker and client-server in the past but decided to duplicate code instead to make it more robust as some users were renaming source code directories. https://github.com/mlcommons/power-dev/pull/198